docs/platform/release-notes/tizen-5-5-m3.md

```diff
@@ -0,0 +1,93 @@
+﻿# Tizen 5.5 Public M3 Release Notes
+
+Release date: Aug. 31, 2020
+
+
+- Tizen 5.5 public M3 adopts some new features for wearable, such as AOD viewer, Sticker framework, Aurum for testing-automation, and Battery-Monitor framework.
+
+
+## Application framework

```

Comment:
There needs to be heading 
##Release Notes 
and other headings should be adjusted accordingly, see previous release notes

Reply:
We haven't used "##Release Notes" before. We used the same current headings for other headings. I think someone added this heading(##Release Notes) regardless of the result of the previous review request. Anyway, do we need to add "##Release Notes" and adjust other headings from this release?

Reply:
@mijin85cho, could you confirm what needs to be done in this case?

Reply:
For the release note, after the "release note" parts were merged, we have been added "Release Details" part separately.

If @wtprince add the "##Release Notes" title and adjust the title depth, it would be helpful for later work indeed. As for now we don't have "Release Details" information, I think modifying "##Release Notes" part only will be helpful enough. 
@wtprince would you?

Reply:
OK. I will.

Reply:
> OK. I will.

Thank you for that.